README.md

```diff
@@ -145,6 +145,7 @@ create an [issue](#contribution).
 
 | Version |    latest - 3.5    |     3.4 - 3.1      |        3.0         |
 |---------|:------------------:|:------------------:|:------------------:|
+| 2024.4  | :heavy_check_mark: |        :x:         |        :x:         |
```

Comment:
With the changes of the license check etep versions <=3.5 are not compatible with ecu.test 2024.4